spot_wrapper/calibration/README.md

```diff
@@ -0,0 +1,242 @@
+in testing for the default distance viewpoint range.
+
+After the calibration is finished, Spot stows it's arm and sits back down. At this point,
+it is safe to take control of Spot from the tablet or ROS 2 , even if the calibration script is still
+running. Just don't stop the script or it will stop calculating the parameters :) 
+
+## Example Usage (a.k.a Hand Specific Live Incantations)
+For all possible arguments to the Hand Specific CLI tool, run ```python3 calibrate_spot_hand_camera_cli.py -h```.
+Many parameters are customizable.
+
+If you'd like to calibrate depth to rgb, with rgb at default resolution, saving photos to ```~/my_collection/calibrated.yaml```, 
+here is an example CLI command template, under the default tag (recommended for first time)
+```
+python3 calibrate_spot_hand_camera_cli.py --ip 10.17.30.IP -u user -pw SECRET --data_path ~/my_collection/ \
+--save_data True --result_path ~/my_collection/calibrated.yaml --photo_utilization_ratio 1 --stereo_pairs "[(1,0)]" \
+--spot_rgb_photo_width=640 --spot_rgb_photo_height=480 --tag default
+```
+If you'd like to load photos, and run the calibration with slightly different parameters, 
+while saving both the resuls and the parameters to same the config file as in the previous example.
+Here is an example CLI command template (from recorded images, no data collection)
+```
+python3 calibrate_spot_hand_camera_cli.py --data_path ~/my_collection/ --from_data \
+--result_path ~/my_collection/bottle_calibrated.yaml --photo_utilization_ratio 2 --stereo_pairs "[(1,0)]" \
+--spot_rgb_photo_width=640 --spot_rgb_photo_height=480 --tag less_photos_used_test_v1
+```
+If you'd like to calibrate depth to rgb, at a greater resolution, while sampling
+viewpoints at finer X-rotation steps relative to the board, and slightly further from the board
+with finer steps, here is an example CLI command template. Also,
+to demonstrate the stereo pairs argument, let's assume that you also want to find rgb to depth (redundant for 
+demonstration purposes), while writing to the same config files as above.
+```
+python3 calibrate_spot_hand_camera_cli.py --ip 10.17.30.IP -u user -pw SECRET --data_path ~/my_collection/ \
+--save_data True --result_path ~/my_collection/calibrated.yaml --photo_utilization_ratio 1 --stereo_pairs "[(1,0), (0,1)]" \
+--spot_rgb_photo_width=1920 --spot_rgb_photo_height=1080 --x_axis_rot_viewpoint_range -10 10 1 \
+--dist_from_board_viewpoint_range .6 .9 .1
+```
+
+## Improving Calibration Quality
+If you find that the calibration quality isn't high enough, try a longer calibration
+with a wider variety of viewpoints (decrease the step size, increase the bounds). 
+The default calibration viewpoint parameters are meant to facilitate a quick  calibration
+even on more inexpensive hardware, and as such uses a minimal amount of viewpoints.
+
+## Using the Registered Information with Spot ROS 2
```

Comment:
Wondering if the details of this section should be moved to spot_ros2. We can definitely point people there in this readme (like -- "if you want a nice integration of this feature with ros2, check out this section on spot_ros2" and link it) but the wrapper is supposed to be ros agnostic. It's also used by the spot ros1 repo, which we do not maintain and won't have a node that does this.

Reply:
@khughes-bdai @tcappellari-bdai 

Yeah that makes sense I have pulled out the ROS 2 info into the main readme